src/cljs/rems/administration/catalogue_items.cljs

```diff
@@ -104,8 +123,8 @@
                      {:td [:td.active
                            [readonly-checkbox {:value checked?}]]
                       :sort-value (if checked? 1 2)})
-           :commands {:td [:td.commands
-                           [to-catalogue-item (:id item)]
+           :commands {:td [:td.commands {:on-click #(.stopPropagation %)}
```

Comment:
Why stopPropagation? A clarifying comment would be necessary.

Reply:
Otherwise clicking on a button link in commands will propagate to row selection.

Reply:
Please add a comment which says that.

Reply:
Clicking the resource/form/workflow links also selects the row, so each of those would also need the same treatment. Or then in the click handler for the row check what was the target element of the click event, and only select the row if it was a `td` element. Or then just rely on the checkboxes for row selection.

Reply:
I think I'll try improving the row click handler. That way the row selection click handling does not propagate to table users.

Reply:
The click handler now checks that the target was a `td`, `th` or `tr` element which I think is good enough for this purpose as the default action.